commands.go

```diff
@@ -277,15 +277,6 @@ func cmdLs(c *cli.Context) {
 
 	for _, host := range hostList {
 		if !quiet {
-			tmpHost, err := store.GetActive()
```

Comment:
The purpose of this code was to trigger the errors, it's not really needed. It's code that was left in an old PR of mine and was merged.